cmd/storj-sim/network.go

```diff
@@ -258,6 +261,48 @@ func newNetwork(flags *Flags) (*Processes, error) {
 	return processes, nil
 }
 
+// networkPayments generates a satellite's payment report for a given period
+func networkPayments(flags *Flags, args []string) error {
```

Comment:
This change shouldn't be needed, you can run payments binary against the network rather than specifying a separate command.